src/schemas/json/dotnet-releases-index.json

```diff
@@ -101,14 +97,12 @@
         "latest-sdk",
         "product",
         "releases.json",
-        "release-type",
         "support-phase"
       ],
       "additionalProperties": false
     }
   },
   "description": "A collection of manifests for .NET products, which is updated with each preview and stable release of the .NET SDK and/or Runtime",
-  "id": "https://json.schemastore.org/dotnet-releases-index.json",
```

Comment:
Any reason this was removed? At at SchemaStore, schemas must have the `id` properly if they are `draft-04` or `$id` if they are any other draft level.